doc/Support/FAQ.md

```diff
@@ -125,6 +125,13 @@ using the [included
 snmpd.conf](https://raw.githubusercontent.com/librenms/librenms/master/snmpd.conf.example)
 file.
 
+If you are behind a Proxy or thru a NAT'd ipaddress with an Ngix, you 
```

Comment:
Typo here Ngix -> Nginx

Reply:
thx

Reply:
Also, can you be more explicit about what scenarios this will help with?

Reply:
Sure, this helps when you have a NAT in front of libre NMS, in my case I'll be doing proxy also, but this is valid for any nginx environment. in apache you have MultiViews , in Nginx you do not. so this is a workaround for that.

If you have an Nginx, and you try to open /device/.../graph.php?args you cannot, and you will see a 404 error.

In this Nginx workaround, what it does is to "try" the "graph.php" in the http_root, where it is in fact. whar way Nginx stripes /device/ part and opens the right "graph.php" in the right path. If not, the 404 was a pointer that nginx was trying to find the "graph.php" inside some path /device/.../.../ that doesnt exists, for that the browser shows a 404 - file not found .

this was my troubleshooting last night, lost some time on this, and this was my fix .

keep up the good work, Librenms is awesome .

Reply:
Yup, performance tab is not showing...

SyntaxError: expected expression, got '<'  vis.min.js:1
ReferenceError: vis is not definedtab=performance:682:22

Reply:
sorry I might have been mistaken, I'm on a docker image running librenms, and still not sure what version I have...
validade shows 1.63, but i still have some files from previous versions...
ex: ${LIBRENMS_PATH}/includes/html/pages/device/performance.inc.php (should not be there)

Reply:
sorry I need to redo this pull request. It's no good this way. sorry